python/pyarrow/conftest.py

```diff
@@ -20,7 +20,6 @@
 from pyarrow import Codec
 from pyarrow import fs
 
-import numpy as np
```

Comment:
I think it would be nice to standardize on the `try/except` snippet that's used in other places.